src/ray/object_manager/plasma/malloc.h

```diff
@@ -49,4 +41,6 @@ struct MmapRecord {
 /// and size.
 extern std::unordered_map<void *, MmapRecord> mmap_records;
 
+/// private function, only used by PlasmaAllocator
+bool GetMallocMapinfo(void *addr, MEMFD_TYPE *fd, int64_t *map_length, ptrdiff_t *offset);
```

Comment:
If it's a get, please mark it as const

Reply:
this is unfortunate a C function. I'll make addr const.